apk-generator/v2/app/static/js/main.js

```diff
@@ -225,6 +233,7 @@ $form.submit(function (e) {
     data.append("email", $emailInput.val());
     data.append("data-source", dataSourceType);
     data.append("build-type", buildType);
+    data.append("is-auth-enabled", isAuthEnabled);
```

Comment:
@iamareebjamal here i have tried `data.append("is-auth-enabled", $authOptionCheckbox.checked);` but don't know why it is not working. Can you please test it?